crates/db_views/report_combined/src/impls.rs

```diff
@@ -273,6 +279,12 @@ impl ReportCombinedQuery {
       if !show_community_rule_violations {
         query = query.filter(filter_admin_reports(Utc::now() - Days::new(3)));
       }
+      // Hide reports of remote communities
+      query = query.filter(
+        report_combined::community_report_id
+          .is_null()
+          .or(community::local),
```

Comment:
We aren't doing this for remote posts or comments, so I don't see why this should be a special case either.

If a federated community gets reported, and resolved, that should get sent out and received, then your below filter should work.

If it doesn't get resolved, then its up to each instance to decide what to do with it, just like comments and posts.

Reply:
I got rid of this filter, so it should now be consistent with posts and comments.